features/lib/Api.js

```diff
@@ -1,38 +1,57 @@
-const axios = require('axios');
+const axios = require("axios");
+const { camelCase } = require('change-case');
+const applyCaseMiddleware = require("axios-case-converter").default;
```

Comment:
Maybe run an autoformatter to get all these quotes consistent.

Reply:
Done!